Tests/OTelTests/Metrics/MetricStore/CounterTests.swift

```diff
@@ -0,0 +1,67 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift OTel open source project
+//
+// Copyright (c) 2024 Moritz Lang and the Swift OTel project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+@testable import OTel
+import XCTest
+
+final class CounterTests: XCTestCase {
+    func testIncrement() {
```

Comment:
I'm happy to perform these changes since they'd be very cumbersome, but in the spirit of adhering to the common naming convention in the library, could we please name these new test methods accordingly? Here, `test_increment` would be more in line with other existing test methods.